### PR TITLE
Feat/add destination form

### PR DESCRIPTION
**Pull request by sebsch1337**
You can add new destinations for the trip now :) please review my code.

### CODE REVIEW

File: components/DestinationItem.js [hunk 1]
```diff
@@ -5,11 +5,18 @@ export default function DestinationItem({ name }) {
   return (
     <ListBox bulletCircle={bulletCircle.src}>
       <BulletCircle src={bulletCircle.src} height={15} />
```
Comment (Mark-Krensel) [inline components/DestinationItem.js:7]: why is the 15 in { }?

**Review by Mark-Krensel — APPROVED** [review]

**Review by anhthu253 — APPROVED** [review]
super cool!

File: components/DestinationItem.js (hunk 1 above)
Comment (sebsch1337) [reply components/DestinationItem.js:7]: opps that shouldnt be there. deleted it :)

File: components/DestinationForm.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
+import styled from "styled-components";
+import Image from "next/image";
+import addButtonCircleActive from "../assets/addButtonCircleActive.svg";
+
+export default function DestinationForm({ onSubmitNewDestination }) {
+  return (
+    <ListBox>
```
[the remark this answers is not in the document]
Comment (sebsch1337) [reply components/DestinationForm.js:7]: Added it - thanks!

File: components/DestinationForm.js [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+      <FormBox onSubmit={onSubmitNewDestination}>
+        <AddTripButton>
+          <Image src={addButtonCircleActive} alt="Add button" />
+        </AddTripButton>
+        <DestinationInput name="destination" required />
```
[the remark this answers is not in the document]
Comment (sebsch1337) [reply components/DestinationForm.js:12]: Also added it here - thanks!

**Review by PhilippWeiss85 — APPROVED** [review]
LGTM!

**Pull request merged**